CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+# Theoretical Chemical Kinetics Database (TCKDB)
```

Comment:
Let's avoid having two same-level titles one after the other

Reply:
Thank you for checking this. I have updated the code of conduct so that the 2 header lines appear as one within the Markdown.